protocol/bin-prot/src/value/layout/traverse.rs

```diff
@@ -45,9 +45,9 @@ impl Iterator for BinProtRuleIterator {
                             rules
                                 .drain(0..)
                                 .map(|field| {
-                                    if field.field_name == "current_protocol_version" {
-                                        println!("{:#?}", field.field_rule);
-                                    }
+                                    // if field.field_name == "current_protocol_version" {
```

Comment:
nit: let's remove this?

Reply:
fixed

Reply:
Disabled this because it prints during benchmark tests, we might need to switch log/trace lib at some point